app/views/comment/create.js.erb

```diff
@@ -1,4 +1,4 @@
-$('#answer-<%= @answer_id %>-comment-section').before('<%= escape_javascript(render :partial => "questions/comment", :locals => {comment: @comment, answer_id: @answer_id}) %>')
+$('#answer-<%= @answer_id %>-comment-section').before('<%= escape_javascript(render :partial => "notes/comment", :locals => {comment: @comment, answer_id: @answer_id}) %>')
```

Comment:
Here too. Because if you want to use `notes/comment` then there is no variable `answer_id` in that partial.

Reply:
Isn't it? @ViditChitkara